contracts/MarginCalculator.sol

```diff
@@ -63,30 +64,40 @@ contract MarginCalculator is Initializable {
         // ensure the long asset is valid for the short asset.
         require(_isMarginableLong(_vault), "MarginCalculator: long asset not marginable for short asset");
 
```

Comment:
need a check to make sure collateral in vault == short.collateral if there is a collateral asset

Reply:
GOOD CATCH!